client/components/Editor/Slides.jsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Sortable from 'react-sortablejs';
 import { Container, Grid, Card, Dropdown, Button } from 'semantic-ui-react';
-import * as Components from '@components/Slide/Components';
 import SlideEditor from '@components/Slide/Editor';
+import SlideList from '@components/SlideList';
```

Comment:
`SlideList` seems like it would be a "slide list" or "list of slides", but it appears to be the list of components in a given slide, right? Can we call this `SlideComponentsList`?

Reply:
Good question, I named it with the same convention as `ScenarioList`, which has the same components list content. Should I change both? I don't mind it either way, just want to be consistent. Also if I do both, maybe that would be better in a follow up PR?

Reply:
I don't think my request applies to `ScenarioList`, which produces a list of Scenarios (via `ScenarioEntries`). Either way, this is not a blocker.